core/modules/system/system.install

```diff
@@ -3391,6 +3391,20 @@ function system_update_1087() {
   }
 }
 
+/**
+ * Enable Admin Bar's Language Switcher component.
+ */
+function system_update_1088() {
+  if (!module_exists('locale')) {
+    return;
+  }
+  $config = config('admin_bar.settings');
+  $components = $config->get('components');
+  $components[] = 'admin_bar.locale';
+  $config->set('components', array_unique($components));
+  $config->save();
```

Comment:
Wondering what the benefit of the `return` is here, since the check is already being performed? 🤔 ...couldn't we simply do this?:
```suggestion
  if (module_exists('locale')) {
    $config = config('admin_bar.settings');
    $components = $config->get('components');
    $components[] = 'admin_bar.locale';
    $config->set('components', array_unique($components));
    $config->save();
  }
```

Reply:
@klonos early returns make it easier to read and follow the logic. See https://stackoverflow.com/questions/38920362/how-is-return-early-concept-useful-in-php and https://freek.dev/1593-writing-readable-php-decrease-indentation-by-returning-early for example.

It's more noticeable if there are number of if statements, but still just fine here.